resources/assets/uad_lists.json

```diff
@@ -18925,5 +18925,196 @@
+    "list": "Oem",
+    "description": "TECNO SPOT. Tecno community app.",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.rlk.weathers",
+    "list": "Oem",
+    "description": "Weathers app.",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.transsion.carlcare",
+    "list": "Oem",
+    "description": "After-sales Service app.",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.talpa.hibrowser",
+    "list": "Oem",
+    "description": "Awful browser loaded with telemetry and ads.",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.transsion.hiparty",
+    "list": "Oem",
+    "description": "Allow to play music simultaneously on multiple devices.",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.talpa.share",
+    "list": "Oem",
+    "description": "XShare Mini. File Sharing App.",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "net.bat.store",
+    "list": "Oem",
+    "description": "AHA Games. Intrusive game ads on HIOS launcher and random popups.",
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.transsnet.store",
+    "list": "Oem",
+    "description": "Palm Store. App store with unsecure apps and probably malware. Shows intrusive ads and popups.",
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.transsion.magicshow",
+    "list": "Oem",
+    "description": "(Bad) Video Player with Ads.",
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.transsion.plat.appupdate",
+    "list": "Oem",
+    "description": "Hogs memory in background.",
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.transsion.magazineservice.hios",
+    "list": "Oem",
+    "description": "Shows news stories and wallpapers on the lockscreen.",
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.idea.questionnare",
+    "list": "Oem",
+    "description": "NEEDS MORE INFO. No effect on uninstall.",
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.transsion.filemanagerx",
+    "list": "Oem",
+    "description": "File Manager App.",
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.gallery20",
+    "list": "Oem",
+    "description": "AI Gallery. Gallery app.",
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
```

Comment:
I don't think that's a good idea to put the default File Manager and Gallery apps in the `Recommended` list, even if those apps are really bad (which I bet they are)

I'd rather put them in the `Advanded` list.

Reply:
So there are by default 2 _File Manager_ apps and 2 _Gallery_ apps on Tecno Phones? Those of the manufacturer and those of Google.

The question is: can we assume it is the case for all the Tecno phones? I'm not sure about that.

Reply:
I improved some descriptions and moved `com.transsion.filemanagerx` and `com.gallery20` to the `Advanced` list.

Your phone has really awful pre-installed apps that's incredible!